eo-maven-plugin/src/test/java/org/eolang/maven/OptimizeMojoTest.java

```diff
@@ -173,7 +173,7 @@ void testSimpleOptimize(@TempDir final Path temp) throws Exception {
     }
 
     @Test
-    void testOptimizeWithFailOnErrorFlag(@TempDir final Path temp) throws Exception {
+    void testWithFailOnErrorFlag(@TempDir final Path temp) throws Exception {
```

Comment:
@MikhailLipanin it's better to rename the test to `failsOnErrorFlag`

Reply:
done